added more details and security-policy sample config

aprabh92 opened the pull request:
Additional details added to detailed readme file and sample config for security policy

aprabh92 commented:
added ipsec example

davedotdev commented on DETAILED_INSTRUCTIONS.md:
```diff
@@ -80,19 +80,21 @@ rm -rf yang
 ### 6.	Generate the YIN and XPath Files
 
-The next step, depending on the size of YANG model/s, may take some time. Prepare some popcorn!
+The next step, depending on the size of YANG model/s, may take some time. Prepare some popcorn! Ensure you have copied all the dependent yang modules into the specified yangDir, else you may run into errors when you run the processYang program. 
```
Can you correct yang to YANG please.

davedotdev commented on DETAILED_INSTRUCTIONS.md:
````diff
@@ -104,6 +106,13 @@ Creating Xpath file: junos-qfx-conf-protocols@2019-01-01_xpath.txt
 Creating Xpath file: junos-qfx-conf-protocols@2019-01-01_xpath.xml
 ```
 
+Note: Copy the below yang files from the common directory as the other junos yang modules such as junos, junos-es, junos-nfx etc are dependent on them
````
Again, please uppercase YANG. It's worth commenting that the dependent YANG files must be from the same set of files. You can't mix and match dependent files from another version, despite the naming being the same.

davedotdev commented on Samples/security_ipsec/interfaces/.terraform/modules/modules.json:
```diff
@@ -0,0 +1 @@
+{"Modules":[{"Key":"","Source":"","Dir":"."}]}
```
Can you please remove the .terraform, .tfstate, .tfstatebackup files. This is local to your environment.

davedotdev commented on Samples/security_policy/yang_files/readme.md:
```diff
@@ -0,0 +1,3 @@
+copy yang files from the below directories of the cloned juniper git repo
```
Please upper case YANG.

davedotdev reviewed (changes requested):
Hey,

Can you remove your .tfstate, .tfstatebackup and .terraform directories please? That local state should not be in examples.

Please also change any instances of lower case yang, to upper case YANG.

Thanks!

aprabh92 commented:
Hello @davedotdev  made the respective changes
- changes based on comments provided
- modified earlier ipsec files with the ones you helped following best practices
- added l3vpn sample configuration
